elesto-dao/elesto
# Chore/update main with v3 rc2

### Description

- cherry-pick commits from v3.0.0-rc2 into main branch

### How to test

- see commits on => https://github.com/elesto-dao/elesto/commits/release/v3.0.0
- ensure last two commits and code match
